README.md

```diff
@@ -23,16 +27,42 @@ lazy val app = (project in file("app"))
   .enablePlugins(AkkaSerializationHelperPlugin)
 ```
 
-## Features
+## Missing serialization binding
 
-### 1. Serializability Checker
+To serialize message, state or event in Akka, Scala trait needs to be defined:
```

Comment:
```suggestion
To serialize message, persistent state or event in Akka, Scala trait needs to be defined:
```

As _state_ alone is a bit unclear/imprecise (I wouldn't think of akka-persistence first)

Reply:
Definitely. It's more meaningful. I would apply this suggestion.